lms/static/styles/components/_BasicLtiLaunchApp.scss

```diff
@@ -12,3 +12,8 @@
   left: calc(50% - #{$spinner-size / 2});
   top: 100px;
 }
+
+.js-via-iframe {
+  /* Fix to allow the sidebar to scroll to the bottom while we have fixed grader header. */
+  padding-bottom: 96px;
```

Comment:
A hack to get the sidebar to be able to scroll to its bottom. The grader is pushing it down and since the sidebar is in an iframe and we use height: 100% it gets cut off. This is just a simple fix, but it's not a great solution because its a calculated padding based on the height of the grader itself. I need to come up with a better css fix.

You'll only see this issue if you have enough annotations to fill up the entire height. Then add a new page note and you won't be able to see the submit button w/o this css fix.

Reply:
Definitely want to get eyes other than mine on this one...gives me pause.